app/docs/enterprise/0.31-x/installation/docker.md

```diff
@@ -55,21 +55,31 @@ A guide to installing locally Kong Enterprise Edition (and its new license file)
           -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
           -e "KONG_VITALS=on" \
           -e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
+          -e "KONG_PORTAL=on" \
+          -e "KONG_PORTAL_API_URI=0.0.0.0:8000/portal" \
+          -e "KONG_PORTAL_GUI_URI=localhost:8003" \
           -e "KONG_LICENSE_DATA=$KONG_LICENSE_DATA" \
           -p 8000:8000 \
           -p 8443:8443 \
           -p 8001:8001 \
-          -p 8002:8002 \
           -p 8444:8444 \
+          -p 8002:8002 \
+          -p 8445:8445 \
+          -p 8003:8003 \
+          -p 8004:8004 \
           kong-ee
 
-10. Starting in Kong `0.30`, the Admin API only listens on the local interface by default. This was done as a security enhancement. If you are already using Kong, and your Admin API still binds to all interfaces, consider updating it as well. You can do so by updating the configuration value, admin_listen like so: `admin_listen = 127.0.0.1:8001`. This has been in the above example with `KONG_ADMIN_LISTEN=0.0.0.0:8001`
+10. Congratulations! You now have Kong Enterprise installed and running. Test it by visiting: http://localhost:8002 (Admin GUI). If you load the Dev Portal (http://localhost:8003) expect a blank page until you follow these instructions: https://getkong.org/docs/enterprise/0.31-x/developer-portal/introduction/
+
+## FAQs
+
+- Starting with Kong `0.30`, the Admin API only listens on the local interface by default. This was done as a security enhancement. Note that here, we are overriding that in the above example with `KONG_ADMIN_LISTEN=0.0.0.0:8001` because Docker container networking benefits from more open settings and enables the Admin GUI & Dev Portal to talk more easily with the Kong Proxy.
```

Comment:
replace
`to talk more easily with the Kong Proxy`
to
`to talk with the Kong Admin API`

Reply:
less words FTW. thank you!